phoebe/atmospheres/passbands.py

```diff
@@ -1678,20 +1394,14 @@ def Imu(self, query_pts, atm='ck2004', ldatm='ck2004', ldint=None, ld_func='inte
         * NotImplementedError: if `ld_func` is not supported.
         """
 
-        if atm not in ['blackbody', 'extern_planckint', 'extern_atmx', 'ck2004', 'phoenix', 'tmap_sdO', 'tmap_DA', 'tmap_DAO', 'tmap_DO', 'tremblay']:
-            raise RuntimeError(f'atm={atm} is not supported.')
-
-        if ldatm not in ['none', 'ck2004', 'phoenix', 'tmap_sdO', 'tmap_DA', 'tmap_DAO', 'tmap_DO', 'tremblay']:
-            raise ValueError(f'ldatm={ldatm} is not supported.')
-
         if ld_func == 'interp':
-            if atm == 'blackbody' and 'blackbody:Inorm' in self.content and ldatm in ['ck2004', 'phoenix', 'tmap_sdO', 'tmap_DA', 'tmap_DAO', 'tmap_DO', 'tremblay']:
+            if atm.name == 'blackbody' and 'blackbody:Inorm' in self.content and ldatm.name in ['ck2004', 'phoenix', 'tmap_sdO', 'tmap_DA', 'tmap_DAO', 'tmap_DO', 'tremblay']:
```

Comment:
Shouldn't we try to avoid these lengthy lists of atmospheres? Perhaps just removing the unwanted atms from _atms_supported?

Reply:
yep, this is WIP. This simplifies to:
```
if ... and hasattr(ldatm, 'mus') ...
```